joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/RubySrc2Cpg.scala

```diff
@@ -40,6 +46,7 @@ class RubySrc2Cpg extends X2CpgFrontend[Config] {
       val astCreationPass = new AstCreationPass(config.inputPath, cpg, global, packageTableInfo)
       astCreationPass.createAndApply()
       TypeNodePass.withRegisteredTypes(astCreationPass.allUsedTypes(), cpg).createAndApply()
+      new ImportResolverPass(cpg, packageTableInfo).createAndApply()
```

Comment:
This should go under post-processing, since "resolving" is usually heuristic based.

Reply:
Agree, holding changing till [this](https://github.com/joernio/joern/pull/3033) gets merged.
The idea is to run `AstPackagePass` in `PostProcessingPass`, because `ImportResolverPass` needs packageTableInfo as a dependency